src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -20,13 +20,20 @@ export type ButtonGroupProps = {
    * and returns the `ReactNode` to render.
    *
    * The placement of this button can be controlled using the `overflowPlacement` prop.
+   *
+   * *Note: this will not work with `orientation='vertical'`.*
```

Comment:
So vertical will require changes in `useOverflow`/`useResizeObserver`. Need opinions.

Should I leave it as is? Should I remove the overflowButton from DOM if vertical is specified and fix it in a future PR? Or should I try to make it work in this same PR?

Reply:
iTwin.js would need vertical button group with overflow functionality. Does it require a lot of changes in our hooks?